ERA/ERAv4.nlogo

```diff
@@ -64,30 +77,50 @@ to-report copy-table [ orig ]
   report copy
 end
 
+
 to setup-ops-nodes
-  foreach n-of 3 base-colors [ c ->   ;; 3 is placeholder for number of jurisdictions
+  let curr-jurisdiction 0
+  set CURRENCY-VALUES (list)
+  while [curr-jurisdiction < num-jurisdictions] [   ;; num-jurisdictions does not include decentralized
     ask one-of patches [
       sprout-ops-nodes 1 [
-        set color c
+        set color grey
         set shape "Circle (2)"
         set size 2
         set node-jurisdiction [who] of self
+        set ERiC  random-normal 1 0.1
+        set CURRENCY-VALUES lput ERiC CURRENCY-VALUES
+        set label word [who] of self "   " ;;spacing for formatting
+        set label-color black
+        set soil-degradation-rate random-normal avg-soil-deg-rate (avg-soil-deg-rate / 5)
+        output-print word word "Jur " ([who] of self)  word "   Soil-Deg-Rate " (precision soil-degradation-rate 5)
+
       ]
     ]
+    set curr-jurisdiction curr-jurisdiction + 1
   ]
-  create-ops-nodes 1 [  ;; this is standing in for the decentralized ops-node (like cryptos)
-    set color white
-    set shape "Circle (2)"
-    set size 2
-    setxy random-xcor random-ycor
-    set node-jurisdiction "decentralized"
+
+  set curr-jurisdiction 0
+  while [curr-jurisdiction < num-decentralized-currencies] [
```

Comment:
I originally created this because I didn't know how to make the decentralized currencies line up in the corner, but I just came up with a solution that uses the "who" of the ops-node